test-runner/test_runner.py

```diff
@@ -64,6 +64,13 @@ def parse_args(args: list):
     parser.add_argument(
         "-l", "--logs", dest="logs_path", default="output", help="-l /path/to/logs"
     )
+    parser.add_argument(
+        "-n",
+        "--name",
+        dest="test_name",
+        help="unique test name identifier",
+        required=True,
```

Comment:
Can we make this optional and create a default value?

Reply:
I have removed this as option as it is no longer needed.